src/settings/LoanPolicy/components/EditSections/RenewalsSection/RenewalsSection.test.js

```diff
@@ -0,0 +1,327 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  within,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import { Field } from 'react-final-form';
+
+import {
+  Checkbox,
+  Select,
+  TextField,
+} from '@folio/stripes/components';
+import Period from '../../../../components/Period';
+import {
+  intervalPeriods,
+  renewFromOptions,
+} from '../../../../../constants';
+import RenewalsSection from './RenewalsSection';
+
+jest.mock('../../../../components/Period', () => jest.fn(() => null));
+jest.mock('../../../../utils/options-generator', () => jest.fn((func, options, id) => ({
+  id,
+  options,
```

Comment:
Looks like we have to add additional returned property in object to test that `optionsGenerator` function is called as we discussed?

Reply:
It's not so necesary, because we already modified passed props into object with our custom keys. Our object will match with passed one without our mocked function only if we check tests and pass expected object on straight.